prisma/schema.prisma

```diff
@@ -37,4 +37,16 @@ model AIConfiguration {
   apiKey            String?
```

Comment:
You will need to store this as an encrypted value.

https://www.npmjs.com/package/prisma-field-encryption

https://stackoverflow.com/questions/73747208/how-can-i-encrypt-and-decrypt-data-with-prisma

https://zenstack.dev/blog/custom-attributes#access-attribute


Zenstack builds on top of Prisma to provide even more functionality.